### DISCUSSION TITLE
卡片 Card（ image	）如何支持URL的图片访问

### DISCUSSION THREAD

**Discussion by dongdianxinNK** [TaleLin/lin-ui#1210] [求助] [answered]
<l-card type="primary" 
          full="{{true}}"
          image="/image/4.png"
          title="做一个热爱生活的人！">
     <view class="content">
      大雄要死掉了,他对妻子说:我死之后你找一个男人我就在地下打一一个滚。
      说完就咽气了。某天...
    </view>
  </l-card>
===================================================================
问题：
image="/image/4.png"，此处我想换成类似www.xx.com/image/4.png 的url,小程序中src属性可以支持URL的形式，但是咱们的image不支持URL的形式，想问一下，如何支持URL形式的图片展示呢？

**Comment by juzi214032** [comment, marked as answer]
`Card` 组件的 `image` 属性是支持 http url 的，参考代码
https://github.com/TaleLin/lin-ui/blob/master/examples/pages/components/layout/pages/card/index.wxml
